modules/core_api/fsw/inc/cfe_evs_api_typedefs.h

```diff
@@ -0,0 +1,67 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/**
+ * @file
+ *
+ *  Title:    Event Services API Application Library Header File
+ *
+ *  Purpose:
+ *	           Unit specification for Event services library functions and macros.
+ *
+ *  Design Notes:
+ *
+ *  References:
+ *     Flight Software Branch C Coding Standard Version 1.0a
+ */
+
+#ifndef CFE_EVS_API_TYPEDEFS_H
+#define CFE_EVS_API_TYPEDEFS_H
+
+/********************************** Include Files  ************************************/
+#include "common_types.h" /* Basic data types */
+#include "cfe_evs_extern_typedefs.h"
+
+/** \name Common Event Filter Mask Values  */
+/** \{ */
+#define CFE_EVS_NO_FILTER        0x0000 /**< \brief Stops any filtering.  All messages are sent. */
+#define CFE_EVS_FIRST_ONE_STOP   0xFFFF /**< \brief Sends the first event.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_TWO_STOP   0xFFFE /**< \brief Sends the first 2 events.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_4_STOP     0xFFFC /**< \brief Sends the first 4 events.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_8_STOP     0xFFF8 /**< \brief Sends the first 8 events.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_16_STOP    0xFFF0 /**< \brief Sends the first 16 events.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_32_STOP    0xFFE0 /**< \brief Sends the first 32 events.  All remaining messages are filtered. */
+#define CFE_EVS_FIRST_64_STOP    0xFFC0 /**< \brief Sends the first 64 events.  All remaining messages are filtered. */
+#define CFE_EVS_EVERY_OTHER_ONE  0x0001 /**< \brief Sends every other event. */
+#define CFE_EVS_EVERY_OTHER_TWO  0x0002 /**< \brief Sends two, filters one, sends two, filters one, etc */
+#define CFE_EVS_EVERY_FOURTH_ONE 0x0003 /**< \brief Sends every fourth event message.  All others are filtered. */
+/** \} */
```

Comment:
Should probably add the explanation of the bitmask in case someone wants to create own (ie if someone wanted FIRST 256 or 2^8, ie what makes the First vs ever other, if it is just sign, could use int16 and then note positive is every x one, and negative is first x

Reply:
https://github.com/nasa/cFE/issues/1390